astropy/utils/report_info.py

```diff
@@ -0,0 +1,84 @@
+r"""
+A helper script to automate system info in bug reports.
+
+This can be run as
+```
+python -c "from astropy.utils.report_info import main; main()"
```

Comment:
This might not work on Debian-based systems (including Ubuntu) because on those Python is by default installed as `python3`, not `python`.

Reply:
So, what are you suggesting ? should I mention it in text or should I just present the 2-line python script as such ?

Reply:
in passing, I'm tempted to go with `python -m astropy.utils.report_info` instead (already implemented)